contrib/freshmaker-cli

```diff
@@ -225,6 +248,49 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, date_from=date.today() - timedelta(7),
+                               date_to=date.today()):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL and the issue date is between specified
+    dates.
+    """
+    if (date_to - date_from).days > 7:
```

Comment:
Why is this restricted?

Reply:
We've discussed with @gnaponie , that it's not necessary find advisories in time period more than 7 days

Reply:
@mprahl The purpose of this feature is to be able to check if FM missed any advisory. When you do this process manually you don't need too back in time. So we figured one week might be enough for this purpose. Do you think we should remove the restriction?

Reply:
If we introduce pagination we can get rid of this restriction.

Reply:
I don't really get the need for a date range. Why not just allow the script to specify how many days to go back and max it out to 7?

Reply:
Imagine that you are debugging some new card (that was filed a couple of days ago), you are interested only to that date range, not all days. But, yeah, it's not a big deal, also the num of days to go back is also good.

Reply:
@gnaponie unless you can filter by a date range in ET, you're not gaining much efficiency by restricting the number of days since you have to go through all the pages to find the page that starts with that date anyways. Since ET sorts the dates (I think that was said earlier in this PR), you could do a slightly modified version of the binary search algorithm, but that seems a little complex for a simple script.

This is why I proposed just having x number of days to go back with a maximum.

Reply:
@mprahl . @gnaponie . PTAL. I've changed it now